packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec

```diff
@@ -30,7 +30,7 @@ BuildRequires: ruby < 4
 BuildRequires: rubygems-devel
 BuildRequires: rubygem(foreman_ansible)
 BuildRequires: rubygem(foreman-tasks)
-BuildRequires: rubygem(katello)
+BuildRequires: rubygem(katello) >= 4.14.0.rc1.1
```

Comment:
```
No matching package to install: 'rubygem(katello) >= 4.14.0.rc1.1'
```

That's not an RPM version definition, but Ruby, and RPM gets confused by that. (`4.14.0.rc1.1` is "bigger" than `4.14.0` which we have).

Reply:
I am guessing I need to a make a new gem with it pinned correctly in the gem spec?

Reply:
Packaging as such doesn't look inside the gem.
So editing it here to just read `>= 4.14.0` should be perfectly fine.
(And then update it in the gem whenever you do the next release)